### ISSUE TITLE
Provide setup.py and __init__.py in installation

### ISSUE THREAD

**Issue by luciansmith** [sbmlteam/libsbml#39]
When we build the python bindings, we create setup.py and __init__.py files in the build/ directory, and use them to install libsbml.

However, the installation of the bindings (install/bindings/python/) doesn't include them.  It shouldn't be too difficult to just copy the existing files into the appropriate directories.

**Comment by fbergmann** [comment]
Actually the last releases have been made out of: 

<https://github.com/sbmlteam/python-libsbml>

where a setup.py file builds libsbml + dependencies for the specific python version (including the security flags that specific python distributions use). Using that setup script, the correct __init__ file will also be generated. So i would argue this is done. Though of course we could always work on better documentation. 

This approach has now also been accepted for use in conda-forge and we have now official packages there with: 

<https://github.com/conda-forge/python-libsbml-experimental-feedstock>
<https://github.com/conda-forge/python-libsbml-feedstock>

**Comment by luciansmith** [comment]
That's great!  Does this happen when you do a 'make install' from libsbml?

**Comment by fbergmann** [comment]
no, it happens on python setup.py build / python setup.py bdist_wheel or any of those.

**Comment by luciansmith** [comment]
So, we should remove the python installation from core libsbml?  As it is, with no __init__.py file, it doesn't work.

**Comment by fbergmann** [comment]
i would not remove it, because it is the basis for the setup.py script. It does work currently. So please don't remove anything.

**Comment by luciansmith** [comment]
The current install does not work--it does not copy the required ```__init__.py``` file to the installation directory.

**Comment by luciansmith** [comment]
And I guess that brings us back to my initial request.  While creating a python-libsbml project is nice and all, if core libsbml is going to claim that it will create python bindings, it should create the python bindings, which means including the ```__init__.py``` file.  And if we're going to fix that, we might as well also copy over the ```setup.py``` file at the same time.
